great_expectations/expectations/core/expect_column_pair_values_to_be_equal.py

```diff
@@ -36,7 +22,7 @@ class ExpectColumnPairValuesToBeEqual(TableExpectation):
         column_B (str): The second column name
 
     Keyword Args:
-        ignore_row_if (str): "both_values_are_missing", "either_value_is_missing", "neither"
+        ignore_row_if (str): "both_values_are_missing", "either_value_is_missing", "never"
```

Comment:
@NathanFarmer Let's leave it as `"neither"` for now, because it matches the documentation: (`https://legacy.docs.greatexpectations.io/en/latest/autoapi/great_expectations/dataset/dataset/index.html#great_expectations.dataset.dataset.Dataset.expect_select_column_values_to_be_unique_within_record`).  Then, let us do a clean-up PR, whereby the directive in the `ExecutionEngine` subclasses will be modified accordingly (`"never"` for `MULTICOLUMN`, `"neither"` for `COLUMN_PAIR`).  Happy to discuss.  Thank you!

Reply:
@NathanFarmer Update: the use of `"neither"` (which is the correct value) is handled in PR #3254 (so only revert the value in the present PR).  Thanks!

Reply:
@alexsherstinsky I changed it back. Note: we will also need to change the tests in `tests/expectations/metrics/test_core.py` to reflect the `ExecutionEngine` change to `neither` for `COLUMN_PAIR`.